### ISSUE TITLE
Alt-text on logo

### ISSUE THREAD

**Issue by sweco-semtsg** [hajkmap/Hajk#1553]
WCAG 1.1.1 (A) 

https://www.digg.se/webbriktlinjer/alla-webbriktlinjer/beskriv-med-text-allt-innehall-som-inte-ar-text

Logo is missing alt-text
